src/data.js

```diff
@@ -1,5 +1,29 @@
 // keep it commonjs export
 module.exports = [
+  {
+    name: 'Rowe Morehouse',
+    description:
+      'Growth · Product · Software Project Management · Frontend Dev · Design · Technical Writing · Sales',
+    url: 'https://rowe-morehouse.github.io/resume/uses/',
```

Comment:
the page don't shows what tools you use, so it don't fit's this projecs

Reply:
Look again, the page has all the tools I use: [https://rowe-morehouse.github.io/resume/uses/](https://rowe-morehouse.github.io/resume/uses/)

Reply:
nice